components/splitter/SplitBar.tsx

```diff
@@ -160,8 +166,22 @@ const SplitBar: React.FC<SplitBarProps> = (props) => {
   };
 
   // ======================== Render ========================
-  const StartIcon = vertical ? UpOutlined : LeftOutlined;
-  const EndIcon = vertical ? DownOutlined : RightOutlined;
+  const [startIcon, endIcon, startCustomize, endCustomize] = useMemo(() => {
```

Comment:
```suggestion
  const [startIcon, endIcon, startCustomize, endCustomize] = React.useMemo(() => {
```
上面不用 import 了，这样有个好处是可以和 rc-util 里面的自定义 useMemo 区分开来